app/services/slack_notifier.rb

```diff
@@ -1,9 +1,9 @@
 class SlackNotifier
-  def initialize
+  def initialize(channel = Settings.slack.channel)
```

Comment:
is this set to injectotron channel?
  
Ahh, you override anyway with cccd_development. perhaps that channel should be the default, but for another time perhaps